cell_classification/post_processing.py

```diff
@@ -0,0 +1,51 @@
+import numpy as np
+import pandas as pd
+
+
+def process_to_cells(instance_mask, prediction):
+    """Process predictions from pixel level to cell level
+    Args:
+        instance_mask (np.array):
+            2D array of instance mask
+        prediction (np.array):
+            2D array of pixel level prediction
+    Returns:
+        np.array:
+            2D array of cell level averaged prediction
+        pd.DataFrame:
+            DataFrame of cell level predictions
+    """
+
+    unique_labels = np.unique(instance_mask)
+    unique_labels = unique_labels[unique_labels != 0]
+    mean_per_cell_mask = np.zeros_like(instance_mask, dtype=np.float32)
+    df = pd.DataFrame(columns=['labels', 'pred_activity'])
+    i = 0
+    for unique_label in unique_labels:
```

Comment:
This can be runtime optimized once we're working with a bigger dataset, for now it's fine.